tf-modules/cumulus/outputs.tf

```diff
@@ -169,3 +169,16 @@ output "ecs_cluster_name" {
 output "start_sf_queue_url" {
   value = module.ingest.start_sf_queue_url
 }
+
+
+# Ingest
+
+output "cumulus_ingest" {
```

Comment:
I'm not certain we're going to want to implicitly expose everything the ingest module is providing over being explicit about the specific outputs, much for the same reasons APIs make deliberate choices to expose outputs.    We don't want to create a situation where refactors (such as making the modules more....modular) result in significant user breaking changes.     I think that's worth a discussion before this is approved.